aspnetcore/test/troubleshoot-azure-iis.md

```diff
@@ -43,6 +43,31 @@ In Visual Studio, an ASP.NET Core project defaults to [IIS Express](/iis/extensi
 
 ::: moniker-end
 
+### 403.14 Forbidden
+
+The app fails to start. The following error is logged:
+
+```
+The Web server is configured to not list the contents of this directory.
+```
+
+The error is usually caused by a broken deployment on the hosting system, which includes any of the following scenarios:
+
+* The app is deployed to the wrong folder on the hosting system.
+* The deployment process failed to move all of the app's files and folders to the deployment folder on the hosting system.
+* The *web.config* file is missing from the deployment, or the *web.config* file contents are malformed.
+
+Perform the following steps:
+
+1. Delete all of the files and folders from the deployment folder on the hosting system.
+1. Redeploy the contents of the app's *publish* folder to the hosting system using your normal method of deployment, such as Visual Studio, PowerShell, or manual deployment:
```

Comment:
`publish` or *publish*? I prefer the former but if y'all have a convention never mind me.

Reply:
Yes ... by convention, folder and file names are italics.

Reply:
call it good, then, mate.